TA_dataset/README/ta_dataset_settings.conf.spec

```diff
@@ -1,5 +1,9 @@
 [account]
 url = <string>
+tenant = <bool>
```

Comment:
here it is `bool`, why do we use `string` in other parts?

Reply:
converted to string